docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,415 @@
+  // Configure the UID2 module with a new token
+}
+
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        uid2Token: {
+          'advertising_token': '...advertising token...',
+          'refresh_token': '...refresh token...',
+          // etc. - see the sample token for contents of this object
+        }
+      }
+    }]
+  }
+});
+```
+
+:::caution
+If you configure a user ID by calling `setConfig` (or any similar function) twice, you will need to call `refreshUserIds` for the user ID submodules, to reinitialize their ID values.
+:::
+
+## Checking the Integration
+
+To check that the UID2 module has a valid UID2 token, call `pbjs.getUserIds().uid2`. If a value is returned, a valid UID2 token still exists in the UID2 module.
+
+If there are problems with the integration, here are some steps you can take:
+
+- Check the browser console logs.
+- Use the browser developer tools to inspect the API calls to the UID2 service.
+
+For additional help, refer to Prebid's documentation on [Troubleshooting Prebid.js](https://docs.prebid.org/troubleshooting/troubleshooting-guide.html) and [Debugging Prebid.js](https://docs.prebid.org/debugging/debugging.html).
+
+An example of a tool for validating and debugging Prebid.js configuration is Professor Prebid, an open-source Chrome extension:
+
+- Chrome web store download location: [Professor Prebid](https://chromewebstore.google.com/detail/professor-prebid/kdnllijdimhbledmfdbljampcdphcbdc)
+- Documentation on prebid.org: [Professor Prebid User Guide](https://docs.prebid.org/tools/professor-prebid.html)
+
+## Configuration Parameters for userSync
+
+The following parameters apply only to the UID2 Prebid User ID Module integration.
+
+In this table, CR = client refresh mode, SO = server-only mode, and N/A = not applicable.
+
+| Param under userSync.userIds[] | Mode/Scope | Type | Description | Example |
+| --- | --- | --- | --- | --- |
+| name | CR: N/A<br/>SO:&nbsp;Required | String | ID value for the UID2 module. Always `"uid2"`. | `"uid2"` |
+| value | CR: N/A<br/>SO: Optional | Object | An object containing the value for the advertising token. | See [Configuration Parameter Examples: Value](#configuration-parameter-examples-value). |
+| params.uid2Token | CR: Optional<br/>SO: N/A | Object | The initial UID2 token. This should be the `body` element of the decrypted response from a call to the `/token/generate` or `/token/refresh` endpoint. | See [Sample Token](#sample-token). |
+| params.uid2Cookie | CR: Optional<br/>SO: N/A  | String | The name of a cookie that holds the initial UID2 token, set by the server. The cookie should contain JSON in the same format as the uid2Token param. **If uid2Token is supplied, this param is ignored.** | See [Sample Token](#sample-token). |
+| params.uid2ApiBase | CR: Optional<br/>SO: N/A | String | Overrides the default UID2 API endpoint. | `"https://prod.uidapi.com"` (the default)|
```

Comment:
this is optional for both CR/SO, maybe link to Environment page http://localhost:3006/docs/getting-started/gs-environments so people know what value can be set here?

Reply:
Fixed value and added link, thx much.